src/main/java/org/openmainframeproject/cobolcheck/features/testSuiteParser/TestSuiteParser.java

```diff
@@ -1,15 +1,18 @@
 package org.openmainframeproject.cobolcheck.features.testSuiteParser;
 
 import org.openmainframeproject.cobolcheck.exceptions.*;
+import org.openmainframeproject.cobolcheck.features.interpreter.StringTokenizerExtractor;
```

Comment:
I am going to ask for some rewriting here. 
TestSuiteParser is a feature in the structure of COBOL Check. 
As general rule, features should not be importing features.
I know there are exceptions to that, but it is not the case for TestSuiteParser.
You should be able to get the values you are looking for, from a services, instead of a feature.
Thank you